indicators/tuya-redirect-4ewqnc.yml

```diff
@@ -0,0 +1,38 @@
+title: Tuya Redirect 4eWQNc
+description: |
+    Detects a phishing landing page for the Colombian bank and credit card issuer Tuya.
+    This was found as a result of this kit being deployed on Replit.
+
+
+references:
+    - https://www.tuya.com.co/
+    - https://urlscan.io/result/026d1d48-ba58-477d-a168-0581ef8b9ebc/
+
+detection:
+
+    title:
+      html|contains:
+        - Portal Transaccional
+
+    css:
+      html|contains|all:
+        - link href="App_Themes/404/bootstrap.min.css" type="text/css"
+        - link href="App_Themes/404/Default1.css" type="text/css"
+        - link href="App_Themes/404/bootstrap.min.css" rel="stylesheet"
+
+    form:
+      html|contains:
+        - form name="aspnetForm" method="post" action="
+
+    telegram:
```

Comment:
Detecting telegram bot token & chat id may not be the best idea, since they can change between different actors using the same kit.